server/utils/forms/inputs/calendarDayInput.ts

```diff
@@ -92,7 +92,7 @@ export default class CalendarDayInput {
         .notEmpty({ ignore_whitespace: true })
         .withMessage(this.errorMessages.yearEmpty)
         .bail()
-        .isInt()
+        .isInt({ max: 9999 }) // sure, dates bigger than this are real dates, but they're not valid here
```

Comment:
suggestion to add this to `calendarDay.ts::isValid()`. We can globally invalidate all years that are greater than 9999.

Reply:
simple unit test would be nice

Reply:
yeh i toyed with this - but i wasn't sure it was apprpriate there. that method is checking if the date itself is valid. dates in 20222 kinda _are_ valid... but we don't want them passed through our form, which is why i ended up putting it here.

Reply:
added unit test